app/models/user.rb

```diff
@@ -153,10 +153,17 @@ def num_rated_mentored_solutions
   end
 
   def mentor_rating
-   @mentor_rating ||=
-      solution_mentorships.where.not(rating: nil).
-                           pluck(Arel.sql("AVG(rating) as r")).
-                           first.to_f.round(2)
+    @mentor_rating ||= begin
+      rating_arr = solution_mentorships.where.not(rating: nil).order(:rating).pluck(:rating)
+      if rating_arr.empty?
+        0.0
+      else
+        five_percent = (rating_arr.length * 0.05).round
```

Comment:
Hey @iHiD,  sorry for the misunderstanding about removing 5% from each side rather then 5%  total.  I have changed this, though I did have a question.  So consider we have a mentor with 20 ratings like in the test.  5% of the mentor’s ratings is 1 ratings.  My question is as we can’t remove 0.5 ratings on each side the current method is rounding to the nearest whole number meaning we are removing 1 from each side or 10% total in this example.  Is this ok? The other option would be to determine which side to remove the single rating from, so either removing the examples lowest or highest rating.  Alternately we could also always round down and in this case remove no ratings.  This would happen in many cases such as 100 ratings where 5% of 100 is 5 and 2.5 removed from each side can’t be so we would either need to round and remove 3 from each side, round down and remove 2 from each side or remove 3 from one side and 2 from another.  What are your thoughts?  If you like the option for removing more from one side then the other which side do you think we would remove more from?

Reply:
@ConSou So I've thought about this a bit and chatted to @ccare about it too, and my thinking now is that we should remove **just** the bottom 5%. That means we don't publish someone in a situation where they have `[4,4,4,4,(lots more 4s),5]` then get another `4`, which removes the `5`. So if you want to change it so we just remove the lowest 5%, then that'd be great :) Let me know if you disagree though!

Reply:
@iHiD, this makes sense. To me the main problem this was trying to solve was good mentors getting a lower rating due to things outside of their control and this solution of removing the bottom 5% seems that it would help with this.  I will make the changes :)